### PR TITLE
[ServiceBus] Remove formatter in samples/readme.md

### PR DESCRIPTION
**Pull request by yunhaoling**
Prevent preview samples with unstable apis being published to official DocsMicrosoft/Samples.

related issue: https://github.com/Azure/azure-sdk-for-js/issues/7651

### CODE REVIEW

File: sdk/eventhub/azure-eventhub/samples/README.md [hunk 1]
```diff
@@ -1,13 +1,3 @@
----
```
Comment (KieranBrantnerMagee) [inline sdk/eventhub/azure-eventhub/samples/README.md:1]: As I mentioned OOB, let's have an issue in our backlog (july milestone probably?) to readd this in so we don't forget it.

Comment (yunhaoling) [reply sdk/eventhub/azure-eventhub/samples/README.md:1]: Issue created: https://github.com/Azure/azure-sdk-for-python/issues/11129

**Pull request merged**